### ISSUE TITLE
improve logo on dark mode

### ISSUE THREAD

**Issue by axelpale** [axelpale/genversion#25]
Black text does not read nice against dark background. Causes a bit accessibility problem, immediate strain for new comers.

![2024-01-02 genversion dark mode bug](https://github.com/axelpale/genversion/assets/992081/4fcc56c2-5043-4f2f-a62d-c4d1f15969c9)

**Comment by alexandercerutti** [comment]
FYI, Github offers two ways to specify a different logo for dark mode and light mode.

Look at them here:

https://github.com/stefanjudis/github-light-dark-image-example

I also applied the second method for the logo in [the readme of a repository of mine](https://raw.githubusercontent.com/alexandercerutti/passkit-generator/master/README.md).


For what concerns the first method, I don't know why that repo says the first way is deprecated, as the blog post doesn't say anything about deprecation 🤔

https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/

**Comment by axelpale** [comment]
Thanks Alexander for bringing this technique to my attention. I would definitely use it in a readme of an app project. However, as a general habit, I am hesitant to include such modern neatness into a small low-level NPM package as genversion, especially when a simple, old-school solution exists (just add some white to the transparent areas). There are wide and wild range of websites and apps that display NPM packages and I want to be sure the logo is visible in most of them regardless the quality (or lack of it) of their markdown parsers. With a larger high-level project I might not be so conservative. Still, I am grateful for the tip.

**Comment by alexandercerutti** [comment]
The technique based on `<picture>` is based on a CSS feature and user settings, so IMHO is not a real issue. However, as you like :)

**Comment by axelpale** [comment]
![5f0](https://github.com/axelpale/genversion/assets/992081/fa718149-29bd-4b00-900b-47252bd14326)

**Comment by axelpale** [comment]
Implemented in v3.2 with a white drop shadow
